apps/comps/rpc/router/index.ts

```diff
@@ -1,7 +1,13 @@
+import type { InferRouterOutputs } from "@orpc/server";
+
 import { router as boost } from "./boost";
+import { router as competitions } from "./competitions";
 import { router as user } from "./user";
 
 export const router = {
   boost,
+  competitions,
   user,
 } as const;
+
+export type RouterOutputs = InferRouterOutputs<typeof router>;
```

Comment:
Here is that inferred type that is very useful on the client side if you need a rpc procedure return type. There is also a `InferRouterInputs` if we ever need it.